### ISSUE TITLE
Indent should allow double indent for arguments of first line of a variable definition

### ISSUE THREAD

**Issue by dantman** [eslint/eslint#4795]
Consider the following:
```js
"use strict";
var b, c, d;

// 0
var a = b,
	a = b(),
	a = b(c),
	a = b({c: d}),
	a = b({
		c: d
	});

// 1
var a = b.c()
		.chain()  // valid
		.chain()  // valid
		.chain(), // valid
	a = b.c()
		.chain()
		.chain()
		.chain();

// 2
var a = b(
	{
		c: d
	});

// 3
var a = b(
		{        // invalid
			c: d // invalid
		}),      // invalid
	a = b(
		{
			c: d
		});

a;
```
```
31:3  error  Expected indentation of 1 tab character but found 2   indent
32:4  error  Expected indentation of 2 tab characters but found 3  indent
33:3  error  Expected indentation of 1 tab character but found 2   indent

✖ 3 problems (3 errors, 0 warnings)
```

In 3 the double indentation should be valid like it is in 2.

**Comment by gyandeeps** [comment]
I think the rule is behaving in an expected way in //3. There might be an issue with the lower half of the //3 example. There has been many discussions surrounding this previously.

**Comment by nzakas** [comment]
Yeah, this is behaving as expected. Looks like 3 is a bit irregular. I have a hunch that such a pattern would be difficult to detect. @gyandeeps ?

**Comment by nzakas** [comment]
It looks like you're using `babel-eslint` as your parser. Can you try using the default parser instead? You can just delete the `parser` from your config file to switch to the default parser. Make sure you configure the [parser options](http://eslint.org/docs/user-guide/configuring#specifying-parser-options) as well.

**Comment by nzakas** [comment]
Oops, sorry about that last comment. What I meant:

Unfortunately, it looks like consensus couldn't be reached on this issue and so I'm closing it. While we wish we'd be able to accommodate everyone's requests, we do need to prioritize. We've found that issues failing to reach consensus after 21 days tend never to reach consensus, and as such, we close those issues. This doesn't mean the idea isn't interesting, just that it's not something the team can commit to.
